src/components/backend-ai-registry-list.ts

```diff
@@ -452,12 +460,12 @@ class BackendAIRegistryList extends BackendAIPage {
   }
 
   _resetRegistryField() {
```

Comment:
Same suggestion here. I think it's not necessary to declare and assign member variables to local variables.

Reply:
I reflected it.